### DISCUSSION TITLE
Request for adding new features for current\all windows

### DISCUSSION THREAD

**Discussion by rediffusion** [mohamedmansour/reload-all-tabs-extension#49] [Ideas]
Hello! 🙂

😒
Please add the feature:
1. Reload only unpinned tabs in all windows.
2. Reload all tabs in the current\all window excluding the active tabs.
3. Reload all tabs current\all window excluding the audio tabs.

All the best❗

**Comment by mohamedmansour** [comment]
:o exclude the audio tab is interesting one!

**Comment by mohamedmansour** [comment]
Fixed in #60
